src/pow.cpp

```diff
@@ -18,13 +18,14 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     if (pindexLast == NULL)
         return nProofOfWorkLimit;
 
-    if (params.fPowAllowMinDifficultyBlocks)
```

Comment:
IMHO without the height check it made sense to simply check for `fPowAllowMinDifficultyBlocks`, but now that there is a hardcoded height I think it would make sense to explicitly check testnet instead of fPowAllowMinDifficultyBlocks since that's the real case of the condition right?